Resources/lang/media/fr/media.php

```diff
@@ -13,6 +13,11 @@
         'width' => 'Largeur',
         'height' => 'Hauteur',
     ],
+    'form' => [
+        'alt_attribute' => 'Attribut alt',
```

Comment:
if it's for users comprehension, it's "Attribut de remplacement".
Otherwise, it's "Attribut alt"